### PR TITLE
fixing provider tag value

### PR DESCRIPTION
**Pull request by i332322**
fix provider tag value.

### CODE REVIEW

**Review by ebensom — APPROVED** [review]

File: internal/avs/external_eval_assistant.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (eea *ExternalEvalAssistant) ProvideTags(operation internal.Operation) []*T
 			TagClassId: eea.avsConfig.LandscapeTagClassId,
 		},
 		{
-			Content:    string(operation.ProvisioningParameters.PlatformProvider),
+			Content:    providerCodeByPlan(operation.ProvisioningParameters.PlanID),
```
Comment (ebensom) [inline internal/avs/external_eval_assistant.go:71]: The `providerCodeByPlan` returns "AZR" for azure plan, however our provider tag requires "Azure". 
You might need to introduce a new mapping function in internal/avs package to map plan ID to provider tag.

**Review by ebensom — CHANGES_REQUESTED** [review]

File: internal/avs/external_eval_assistant.go (hunk 1 above)
Comment (i332322) [reply internal/avs/external_eval_assistant.go:71]: ah youre right. i was not thinking about this. aws and gcp it works but not for azure.

**Comment by i332322** [comment]
currently this tagclass allows 3 provider like this. hence the code change to align with it.

image

**Comment by i332322** [comment]
or the other way is make tag class value align with providerCodeByPlan: 
image

File: internal/avs/external_eval_assistant.go (hunk 1 above)
Comment (i332322) [reply internal/avs/external_eval_assistant.go]: ive made my changes.

**Review by ebensom — APPROVED** [review]

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**